omise.js/form-example/index.html

```diff
@@ -16,22 +16,23 @@
 
   <div class="container">
 
-    <form id="card-form" class="card-form">
+    <!-- NOTE: Using `data-name` because we don't want to send any credit card information to the server.  -->
+    <form id="checkout-form" class="checkout-form" action="/" method="GET">
```

Comment:
@ratchagarn can you add `action="/checkout.php" method="POST"`? Just to make an example more explicit on 'what's next' after submit the form.

btw, for note, how about 
```html
<!-- 
NOTE: Using `data-name` to prevent sending credit card information fields to the backend server via HTTP Post
(according to the security best practice https://www.omise.co/security-best-practices#never-send-card-data-through-your-servers).
-->
```